pkg/vendored/opentofu/README.md

```diff
@@ -0,0 +1,3 @@
+# opentofu
+
+Code vendored from [opentofu](https://github.com/opentofu/opentofu). Do not edit, use `go generate` to update.
```

Comment:
Should we add a CI check for vendored code to avoid any accidents here? Run go generate, check repo is clean. Fine as a follow-up.